test/camunda-cloud/features/properties-provider/PropertiesProviderSpec.js

```diff
@@ -95,6 +95,44 @@ describe('camunda-cloud/features - PropertiesProvider', function() {
       shouldHaveGroup(container, 'general', 'multiInstance');
     }));
 
+    it('should not show message props', inject(function(selection, elementRegistry) {
```

Comment:
One minor thing (and you did that already in other places), let's leave two empty lines between test cases. C.f.: https://github.com/bpmn-io/bpmn-js/blob/develop/.github/CONTRIBUTING.md#adhere-to-the-unit-test-style